### DISCUSSION TITLE
What's the correct way to make ElButton function like a link?

### DISCUSSION THREAD

**Discussion by LondonAppDev** [element-plus/element-plus#11324] [Q&A] [answered]
What's the right way to make an ElButton component function the same way a link would?

So for example, I imagine something like this:

```
<el-button href="mailto:someone@example.com">Contact</el-button>
```

Or having a target:

```
<el-button href="http://example.com" target="_blank">Click me</el-button>
```

I know I can achieve this by using raw JS to do something like `window.location = 'http://example.com'` but it feels like a hacky solution.

**Comment by xiyuvi** [comment]
<el-button><a href="mailto:someone@example.com"></a> </el-button>

**Comment by ryuhangyeong** [comment, marked as answer]
## Suggest

Essentially the href attribute is available on the a tag. el-button cannot achieve href effect using button tag.

How about using el-link?

**Reply by lk101** [reply]
+1
